.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "rust/chain-libs"]
```

Comment:
Probably removing this dependency means we should credit IOHK some way for this lib since it now contains a lot of their code.  The code is dual-licensed under MIT & Apache2 so we just have to give credit while respecting the license

Reply:
Added credits to the README